util/roundedFloat.go

```diff
@@ -0,0 +1,9 @@
+package util
+
+import "strconv"
+
+type RoundedFloat4 float64
+
+func (r RoundedFloat4) MarshalJSON() ([]byte, error) {
+	return []byte(strconv.FormatFloat(float64(r), 'f', 4, 32)), nil
```

Comment:
Shouldn't we use 64 bit precision (float64), or we just don't care about rounding issues and we are done here ? :)

Reply:
Well spotted